<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectProperty</class>
 <widget class="QDialog" name="ProjectProperty">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Project property</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="1" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="m_buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QTreeWidget" name="m_tree">
     <property name="minimumSize">
      <size>
       <width>180</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>180</width>
       <height>16777215</height>
      </size>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QStackedWidget" name="m_pages">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="m_projectDefinitionPage">
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Project &amp;Name :</string>
         </property>
         <property name="buddy">
          <cstring>m_nameLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="m_nameLineEdit">
         <property name="whatsThis">
          <string>This is the project name showed in all the application instead of the projet path. Choose a name that can be used to easily find your projet.</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Project Directory :</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Project &amp;RCS :</string>
         </property>
         <property name="buddy">
          <cstring>m_projectRCSComboBox</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="m_projectRCSComboBox">
         <property name="whatsThis">
          <string>This the list of &lt;i&gt;revision control system&lt;/i&gt;. The list is appended by modules that you can find in the customize dialog.</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="_projectLineEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_preloadedFilesPage">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>&amp;Actually preloaded files</string>
         </property>
         <property name="buddy">
          <cstring>m_preloadedFiles</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="m_preloadedFiles">
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_directoryPage">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>&amp;Link to directory :</string>
         </property>
         <property name="buddy">
          <cstring>linkedDirectoryList</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="XinxDirectoryListWidget" name="linkedDirectoryList" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="defaultVisible" stdset="0">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>&amp;Exclude path (use regular expression) :</string>
         </property>
         <property name="buddy">
          <cstring>excludeRegexpPath</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="XinxListWidgetImpl" name="excludeRegexpPath" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="defaultVisible" stdset="0">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>XinxListWidgetImpl</class>
   <extends>QWidget</extends>
   <header location="global">xinxlistwidgetimpl.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>XinxDirectoryListWidget</class>
   <extends>QWidget</extends>
   <header location="global">xinxdirectorylistwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_buttonBox</tabstop>
  <tabstop>m_pages</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>m_buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ProjectProperty</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>492</x>
     <y>375</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>395</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ProjectProperty</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>545</x>
     <y>375</y>
    </hint>
    <hint type="destinationlabel">
     <x>568</x>
     <y>396</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
